test/doctool/test-doctool-html.js

```diff
@@ -9,7 +9,7 @@ try {
 }
 
 const assert = require('assert');
-const { readFile } = require('fs');
+const { readFile } = require('fs/promises');
```

Comment:
This doesn't need promises.

```suggestion
const { readFileSync } = require('fs');
```